substrate/client/offchain/src/api/http.rs

```diff
@@ -48,20 +51,28 @@ use std::{
 
 const LOG_TARGET: &str = "offchain-worker::http";
 
+pub type Body = BoxBody<hyper::body::Bytes, hyper::Error>;
+
+type Sender = mpsc::Sender<Result<hyper::body::Frame<hyper::body::Bytes>, hyper::Error>>;
+type Receiver = mpsc::Receiver<Result<hyper::body::Frame<hyper::body::Bytes>, hyper::Error>>;
+
 /// Wrapper struct used for keeping the hyper_rustls client running.
 #[derive(Clone)]
-pub struct SharedClient(Arc<Lazy<HyperClient<HttpsConnector<client::HttpConnector>, Body>>>);
+pub struct SharedClient(
+	Arc<Lazy<HyperClient<HttpsConnector<client::connect::HttpConnector>, Body>>>,
+);
 
 impl SharedClient {
 	pub fn new() -> Self {
 		Self(Arc::new(Lazy::new(|| {
 			let connector = HttpsConnectorBuilder::new()
-				.with_native_roots()
+				.with_provider_and_native_roots(rustls::crypto::aws_lc_rs::default_provider())
+				.expect("Failed to build HttpsConnector with native roots")
```

Comment:
return error here instead of expect

Reply:
That would make a breaking change. Would it be okay?

Reply:
yes